src/subida.py

```diff
@@ -18,47 +18,15 @@
     UTIL_COLS_COMAFI,
 )
 from risk_data import risk_data
+from clean_numbers import clean_numbers
 from data_info import GenerateDataInfo
 from prepare_comafi_accounts import prepare_comafi_accounts
 from write_data_osiris import Escribir_Datos_Osiris
 
 
 def limpiar_numeros(df_num):

```

Comment:
Why do we keep this function instead of just calling `clean_numbers(df_num)`?

Reply:
If you mean why "limpiar_numeros()" is still there, (which will return the result of clean_numbers()), it´s because I noticed that every function in subida.py is in spanish, and for a matter of consistency I thought it was appropriate that way. However, we can change it, and male limpiar_numeros() dissapear. It may be better to discuss this in the next stand-up?

Reply:
Already changed it, and used `clean_numbers()` instead of `limpiar_numeros()`. Marking as resolved.